sdk/cognitiveservices/Knowledge.QnAMaker/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 2.0.0 (Unreleased)
+  Remove preview tag from the release version
```

Comment:
@chidozieononiwu is the format here the right one or should we remove `Unreleased` ?

Reply:
The format is fine. If you are intending to release the package then the date should be used i.e. `(2020-07-16)` rather than `(Unreleased)`

Reply:
Added date as per suggestion